### PR TITLE
MNT Add behat tests

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripeltd/product-issues/issues/473

### CODE REVIEW

**Review by maxime-rainville — CHANGES_REQUESTED** [review]
Build is still failing on travis: https://app.travis-ci.com/github/silverstripe/silverstripe-totp-authenticator/builds/241725768

**Comment by emteknetnz** [comment]
This is failing in travis though behat passed in github actions :-/  No screenshot in travis means I have no idea what's going on.  @maxime-rainville I think it may be worth merging this regardless so that we actually get a screenshot in travis once it's on the next minor branch

Fail
https://app.travis-ci.com/github/silverstripe/silverstripe-totp-authenticator/jobs/548053589#L1615

Pass
https://github.com/silverstripe/silverstripe-totp-authenticator/runs/4206674802?check_suite_focus=true#step:12:139

**Comment by maxime-rainville** [comment]
I ran the behat test locally and was getting the same error we're seeing on Travis until I added a `SS_MFA_SECRET_KEY` value to my environment. I couldn't find any reference to `SS_MFA_SECRET_KEY` in the raw travis log, so that's probably what's wrong.

File: .travis.yml [hunk 1]
```diff
@@ -1,4 +1,11 @@
 version: ~> 1.0
 
 import:
-  - silverstripe/silverstripe-travis-shared:config/provision/standard-jobs-range-npm.yml
+  - silverstripe/silverstripe-travis-shared:config/provision/standard-jobs-range-behat-npm.yml
+
+env:
+  global:
+    - REQUIRE_RECIPE_TESTING="^2"
+    - REQUIRE_FRAMEWORKTEST="^0.4.2"
```
Comment (maxime-rainville) [inline .travis.yml:9]:
```suggestion
    - REQUIRE_FRAMEWORKTEST="^0.4.2"
    - SS_MFA_SECRET_KEY=TEST123
```

**Review by maxime-rainville — APPROVED** [review]
LGTM

**Pull request merged**